vision/realsense/read_bag.py

```diff
@@ -0,0 +1,113 @@
+"""
+Test program from the examples in the RealSense repo (at
+https://github.com/IntelRealSense/librealsense/tree/master/wrappers/python/examples)
+to read .bag files into a python program
+
+For the program to work, you must enter the correct resolution width, height,
+framerate, depth stream type, and color stream type
+"""
+
+
+# First import library
+import pyrealsense2 as rs
+# Import Numpy for easy array manipulation
+import numpy as np
+# Import OpenCV for easy image rendering
+import cv2
+# Import argparse for command-line options
+import argparse
+# Import os.path for file path manipulation
+import os.path
+
+def read_from_bag(width, height, framerate, depth_type, color_type):
+    # Create object for parsing command-line options
+    parser = argparse.ArgumentParser(description="Read recorded bag file and display depth and color streams in\
+                                    Remember to change the stream resolution, fps and format to match the recorded.")
+    # Add argument which takes path to a bag file as an input
+    parser.add_argument("-i", "--input", type=str, help="Path to the bag file")
+    # Parse the command line arguments to an object
+    args = parser.parse_args()
+    # Safety if no parameter have been given
+    if not args.input:
+        print("No input parameter has been given.")
+        print("For help type --help")
+        exit()
```

Comment:
Parser stuff should go in the if __name__ == '__main__' and the data form it should be passed into the function.